addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -82,7 +82,7 @@ The Kodi thing supports the following channels:
+| systemcommand         | String               | This channel allows to send system commands to `shutdown`, `suspend`, `hibernate`, `reboot` or `quit` Kodi (channel's state options contains available system commands)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
The value in the XML starts with capitals, should that be aligned?

Reply:
Yes, nice idea. Thanks. I will change them.